DataFormats/L1Trigger/src/Muon.cc

```diff
@@ -102,25 +122,49 @@ l1t::Muon::setTfMuonIndex(int index)
   tfMuonIndex_ = index;
 }
 
-void 
+void
+l1t::Muon::setHwEtaAtVtx(int hwEtaAtVtx)
```

Comment:
hi @rekovic - these should all eventually be moved to the header or otherwise inlined...